File: tests/benchmarks/cases/dispatch_loops/dotnet/benchmark_case.cs
namespace BenchmarkRunner;

internal static class DispatchLoopsCase
{
    public const string Name = "dispatch_loops";
    public const string PassBanner = "BENCH_DISPATCH_LOOPS_PASS";

    public static long Run(int scale) => BenchmarkSupport.DispatchLoops(scale);
}

File: tests/benchmarks/cases/dispatch_loops/java/benchmark_case.java
final class DispatchLoopsCase {
    static final String NAME = "dispatch_loops";
    static final String PASS_BANNER = "BENCH_DISPATCH_LOOPS_PASS";

    private DispatchLoopsCase() {}

    static long run(int scale) {
        return BenchmarkSupport.dispatchLoops(scale);
    }
}
